### ISSUE TITLE
Slightly incorrect PHP code generated for cast()

### ISSUE THREAD

**Issue by Jens-G** [HaxeFoundation/haxe#5288]
The following code should print `c = -1` and it does so for [JavaScript](http://try.haxe.org/#e23B5) and C++ targets. But with PHP, it prints `c = 1` instead, which is wrong.

```
class Test {
    static function main() {
        var n : UInt = 3;
        var c = -cast(n & 1,Int);
        trace('c = $c');
    }
}
```

The reasons seems to be, that the generated PHP code lacks braces:

```
$c = -$n & 1;
```

Due to the operator precedence, the - sign is interpreted too early. [Adding braces fixes the issue](http://codepad.org/I0QgOJz0):

```
<?php
$n = 3;
$c = -$n & 1;
$d = -($n & 1);
echo "c = $c\nd = $d";
```

yields this output:

```
c = 1
d = -1

```

(For those who wonder what the heck the use case yould be ... this is part of [decoding zigzag](http://stackoverflow.com/questions/2210923/zig-zag-decoding), so yes, there is one)

**Comment by Simn** [comment]
This looks like an easy fix, but given that it's PHP and not a regression I'm not going to take the risk of addressing it for 3.3.

**Comment by RealyUniqueName** [comment]
This issue is fixed via d5be8c5ac190ea67646641cd84916432aede28b8
